hw/top_earlgrey/ip/xbar/doc/checklist.md

```diff
@@ -86,14 +84,14 @@ Documentation | [NEW_FEATURES_D3][]     | N/A         |
 RTL           | [TODO_COMPLETE][]       | Done        | Resolved: [#837][]
 Code Quality  | [LINT_COMPLETE][]       | Done        |
 Code Quality  | [CDC_COMPLETE][]        | N/A         |
-Review        | [REVIEW_RTL][]          | Not Started | 1st @tjaychen / 2nd @martin-lueker
+Review        | [REVIEW_RTL][]          | Done        | 1st @tjaychen / 2nd @martin-lueker
 Review        | [REVIEW_DELETED_FF][]   | N/A         |
 Review        | [REVIEW_SW_CSR][]       | N/A         |
 Review        | [REVIEW_SW_FATAL_ERR][] | Done        |
 Review        | [REVIEW_SW_CHANGE][]    | N/A         |
 Review        | [REVIEW_SW_ERRATA][]    | Done        |
-Review        | Reviewer(s)             |             |
-Review        | Signoff date            |             |
+Review        | Reviewer(s)             | Done        | @sjgitty @weicaiyang @tjaychen
```

Comment:
I can't be on there, I was physically present. :) Tim and Weicai should suffice.